yang/ietf-l2vpn-ntw.yang

```diff
@@ -1669,23 +1669,29 @@ module ietf-l2vpn-ntw {
                       container rewrite {
                         description
                           "Set the tag rewriting policy for this
-                           EFP";
+                           vpn-network-accesses. Enables the flexible 
+                           rewrites of the layer-2 frame header for data frames 
+                           as they are processed on the vpn-network-accesses.";
                         choice tag-choice {
                           description
                             "Select the tag rewriting policy for this
-                             EFP";
+                             vpn-network-accesses. It defines a set of standard 
+                             tag manipulations that allow for the insertion, 
+                             removal, or rewrite of one or two 802.1Q VLAN tag.s";
                           leaf pop {
                             type enumeration {
                               enum 1;
```

Comment:
what is the meaning of these enums? also, please add a description for each of them